### PR TITLE
Update test cluster build instructions

### PR DESCRIPTION
**Pull request by stefsuse**
- now includes a step-by-step guide from fresh OS install to build

### CODE REVIEW

**Comment by stefsuse** [comment]
@kiall  @ereslibre  
I can't add reviewers or assignees to PRs in this project so pinging here for awareness.

File: caasp-kvm/README.md [hunk 1]
```diff
@@ -9,55 +9,81 @@ changes really will work in the final product.
 
 ## Requirements
 
-In order to run `caasp-kvm`, a system must have `kvm` installed, `libvirtd` and
-`docker` daemons running, and a few additional dependencies must be met. As
-root, run:
+`caasp-kvm` requires several dependencies to build a cluster. You need to install 
+a handful of packages and have a set of daemons/services running.
```
Comment (mjura) [inline caasp-kvm/README.md]: Please specify hardware, system and package requirements or remove this section at all.

**Comment by kiall** [comment]
@stefsuse I've invited you to the kubic-project github org :)

**Comment by stefsuse** [comment]
@mjura Removed the sentence 
also cleaned up commit messages

**Comment by stefsuse** [comment]
@kiall  any idea what might be causing multiple build failures? It always fails at the bootstrap phase: 

Failures:
  1) Download Kubeconfig User downloads the kubeconfig file
     Failure/Error: expect(page).to have_text("Log in to Your Account")
       expected to find text "Log in to Your Account" in "504 Gateway Time-out The server didn't respond in time."
     # ./spec/features/03-download-kubeconfig.rb:32:in `block (2 levels) in <top (required)>'
     # ./spec/spec_helper.rb:94:in `block (3 levels) in <top (required)>'
     # ./spec/spec_helper.rb:94:in `block (2 levels) in <top (required)>'

**Comment by kiall** [comment]
@stefsuse that looks like a random failure - I've retriggered the CI job.

(You can DIY that with the rerun button at top right of the page)

**Comment by stefsuse** [comment]
I'm going to close this PR and issue a new one to try to fix the build.

**Pull request closed without merging**